frontend/src/pages/LightsPage.js

```diff
@@ -1,26 +1,85 @@
 import PageLayout from "../components/PageLayout";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
 import styled from "styled-components";
 import {Link} from "react-router-dom";
 import Button from "@material-ui/core/Button";
+import SaveIcon from "@material-ui/icons/Save";
+import {FormControlLabel, Slider, Switch} from "@material-ui/core";
 
 export default function LightsPage() {
+    //just a basic Layout, it has 0 functionality besides looking ok
     return (
         <PageLayout>
             <Header title="Connect My Hub"/>
             <Wrapper>
-                <h1>This is a PlaceHolder</h1>
-                <p>Whatever, whenever</p>
-                <p>We´re meant to be together</p>
-                <p>I´ll be there, and you´ll be near</p>
-                <p>And that´s the deal, my dear</p>
-                <Link to ="/home"><Button color="primary" variant="contained">Back</Button></Link>
+                <Button variant="contained" color="primary"
+                        size="large" startIcon={<SaveIcon/>}>
+                    <Link class="Links" to="/addlights">Add Lights Device</Link>
+                </Button>
+                <LightsWrapper>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 1"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 2"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 3"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 4"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 3"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
+                    <Light>
+                        <FormControlLabel control={<Switch color="primary"/>} label={"Device 4"}
+                                          labelPlacement="top"/>
+                        <Slider size="small" orientation="vertical"/>
+                    </Light>
```

Comment:
Kommt da noch ne for each, oder sind das fix 4?

Reply:
da kommt eines Tages eine foreach, aber erst backend. Wollte sehen wies ausschaut mit 4 Lichter. Ist nur eine template